pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -871,7 +899,7 @@ func (r *ReconcileSyncSetInstance) setApplySyncConditions(resourceSyncConditions
 		// TODO: we cannot include the actual error here as it currently contains a temp filename which always changes,
```

Comment:
This TODO should get removed or replaced with a comment describing why the filter is needed.

Reply:
Removed.